committee/timetableoperator.go

```diff
@@ -50,6 +51,7 @@ func NewTimeTableOperator(tableName string, driverName DRIVERTYPE) *TimeTableOpe
 		heightQuery:         fmt.Sprintf("SELECT MAX(height) FROM %s WHERE ? >= time AND EXISTS (SELECT * FROM %s WHERE ? <= time)", tableName, tableName),
 		mintTimeQuery:       fmt.Sprintf("SELECT time FROM %s WHERE height = ?", tableName),
 		tipHeightQuery:      fmt.Sprintf("SELECT MAX(height) FROM %s", tableName),
+		timeLayout:          "2006-01-02 15:04:05-07:00",
```

Comment:
this is the timestamp field layout in the `poll.db`, use it to correctly read/write/parse timestamp